docs/UserGuide.md

```diff
@@ -14,118 +14,185 @@ Fine\$$e is a **desktop app for managing finances, optimized for use via a Comma
+</div>
 
-#### Delete an expense: `rm-expense` / `rme`
+### Viewing help: `help`
+Shows a message with instructions on how to access the user guide.
 
-Deletes a specified expense from the finance tracker.
+Format: `help`
 
-Format: `rm-expense INDEX` / `rme INDEX`
+### Add an expense: `add-expense` / `adde` / (on Expense tab) `add`
+Adds an expense to the finance tracker.
 
- - Deletes a specified expense at the specified `INDEX` in the finance tracker.
- - The index refers to the index number shown in the displayed expense list.
- - The index **must be a positive integer** 1, 2, 3, ...
+Format: `add-expense t/TITLE a/AMOUNT d/DATE [c/CATEGORY]...`
 
-Examples:
- - `rm-expense 3` will delete the 3rd expense stored in the finance tracker.
- - `rme 2` will delete the 2nd expense stored in the finance tracker.
+* `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
+* `DATE` should be in `dd/mm/yyyy` format.
 
- <div style="page-break-after: always;"></div>
+Examples:
+* `add-expense t/Bubble Tea a/5 d/03/10/2020 c/Food & Beverage`
+* `adde t/Taxi Home from School a/$13.50 d/10/10/2020 c/Transport c/School`
 
-### 2. Income
+### Add an income: `add-income` / `addi` / (on Income tab) `add`
+Adds an income to the finance tracker.
 
-#### Add income: `add-income` / `addi`
-Adds a new income to the tracker.
+Format: `add-income t/TITLE a/AMOUNT d/DATE [c/CATEGORY]...`
 
-Format: `add-income d/DESCRIPTION a/AMOUNT`
+* `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
+* `DATE` should be in `dd/mm/yyyy` format.
 
 Examples:
-- `add-income d/Internship a/1200` will add a new income with description “Internship” and amount $1200.00.
-- `addi d/Random Angpao a/10` will add a new income with description “Random Angpao” and amount $10.00.
+* `add-income t/Internship a/560 d/03/10/2020 c/Work`
+* `addi t/Angpao money a/$20 d/10/10/2020 c/CNY c/Gift`
 
+### List transactions: `list`
 
-#### List all income: `ls-income` / `lsi`
+Shows a list of all transactions on the current tab.
 
-Shows a list of all income in the finance tracker.
+Format: `list`
 
-Format: `ls-income` / `lsi`
+* On Overview tab: lists all transactions (expenses/incomes).
+* On Expenses tab: lists all expenses.
+* On Income tab: lists all incomes.
 
-#### Delete income: `rm-income` / `rmi`
+### Finding transactions by keyword: `find`
 
-Deletes a specified income in the finance tracker.
+Finds transactions with titles that contain any of the given keywords on the current tab.
 
-Format: `rm-income INDEX` / `rmi INDEX`
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
- - Deletes the specified income entry from the finance tracker.
- - The index refers to the index number shown in the displayed income list.
- - The index **must be a positive integer** 1, 2, 3, ...
+* On Overview tab: searches all transactions (expenses/incomes).
+* On Expenses tab: searches all expenses.
+* On Income tab: searches all incomes.
```

Comment:
Perhaps bullet points detailing the behaviour of the command on each tab could be moved from the format section to the description section? @yongping827 what do you think?